Fix typo

Emilv2 opened the pull request:
## Description:

Fix a typo in ICON, char-histogram does not exists, I assume it should have been chart-histogram.

Emilv2 committed:
Fix typo

Emilv2 committed:
Fix typo

homeassistant commented:
Hello @Emilv2,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`3cb2c2342a0b62be0b61a1a22f90bccf45fd15ad`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;24217&#x2F;commits&#x2F;3cb2c2342a0b62be0b61a1a22f90bccf45fd15ad) - This commit has something that looks like an email address (emil@vanherp.me). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

pvizeli committed:
Don't follow redirect on ingress itself (#24218)

* Don't follow redirect on ingress itself

* Fix comment

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

balloob committed:
Instantiate lock inside event loop (#24203)

balloob committed:
Use resource for index routing. (#24223)

Kane610 committed:
Axis - Handle Vapix error messages (#24215)

balloob committed:
Add manifest support for homekit discovery (#24225)

* Add manifest support for homekit discovery

* Add a space after model check

* Update comment

pnbruckner committed:
Add control of Amcrest indicator light (#23986)

Enable feature by default but allow it to be disabled by "control_light: false" in config.

Get brand from camera instead of assuming Amcrest (since this works with other cameras, too.)

Retrieve RTSP URL in update method instead of in stream_source property and in handle_async_mjpeg_stream method.

Move amcrest imports from methods to global.

pvizeli committed:
Update azure-pipelines.yml for Azure Pipelines

Adminiuga committed:
ZHA requirements version bump. (#24228)

* ZHA requirements version bump.
* zha-quirks version bump.

Kane610 committed:
Don't allow more than one config flow per discovered Axis device (#24230)

soldag committed:
Bump pychromecast (#24234)

balloob committed:
Log HomeKit model (#24229)

balloob committed:
Add GPSLogger device_info and unique_id (#24231)

balloob committed:
GeoFency unique ID and device info (#24232)

rytilahti committed:
add a deprecation warning for tplink device_tracker (#24236)

* add a deprecation warning for tplink device_tracker

* reword the warning a bit

balloob committed:
Mobile app to use device tracker config entry (#24238)

* Mobile app to use device tracker config entry

* Lint

* Re-use device_info

* Lint

balloob committed:
Update home zone when core config updated (#24237)

* Update home zone when core config updated

* Lint

balloob committed:
Do not use the cache dir for PIP installs (#24233)

Emilv2 committed:
fix gitlab_ci sad icon

Emilv2 committed:
Merge branch 'dev' of github.com:Emilv2/home-assistant into dev

homeassistant commented:
Hi @Emilv2, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant#24217).

Once you do that we will be able to review and accept this pull request.

Thanks!

Emilv2 commented:
@balloob What's wrong with this apart from the other commits appearing here? I tried to fix my branch it but it doesn't seem to show up :/ Should I open a new pull request?

robbiet480 commented:
@Emilv2 You need to rebase onto the latest version of the dev branch. You should be able to accomplish that inside of your current branch just by running `git fetch && git rebase dev`.

Emilv2 commented:
Yes I did that, my branch is now just 1 commit ahead of the dev branch but it still shows 23 commits in this pull request. Anything else I need to do?

robbiet480 commented:
You need to run `git push`.

Emilv2 commented:
Yes I pushed, https://github.com/Emilv2/home-assistant shows correctly it's just one commit ahead and when I try to open a new pull request it also shows correctly.

robbiet480 commented:
@Emilv2 Appears to be a GitHub bug, I've checked your commit in this PR against the branch you just linked and they are different commits, so please submit a new PR and abandon this one.

Emilv2 commented:
@robbiet480 Ok, thanks for the help!
